apps/storybook/src/ProfilePage.stories.tsx

```diff
@@ -150,7 +150,7 @@ export const AboutTabEditBiography = () => (
   </ProfilePage>
 );
 
-export const AboutTabPersonalInfoModal = () => (
+export const AboutTabEditPersonalInfo = () => (
```

Comment:
naming consistency with the other profile modal, which in turn was named after the first modal (team tools)